halo2-base/src/utils.rs

```diff
@@ -291,6 +353,8 @@ pub mod fs {
     };
     use rand_chacha::{rand_core::SeedableRng, ChaCha20Rng};
 
+    /// Reads the srs from a file found in "./params/{dir}/kzg_bn254_{k}.srs"
```

Comment:
It's either `./params/kzg_bn254_{k}.srs" or "{dir}/kzg_bn254_{k}.srs" if `PARAMS_DIR` env var is specified.